src/data/multiset/antidiagonal.lean

```diff
@@ -67,6 +67,19 @@ quotient.induction_on s $ λ l, begin
   {congr; simp}, {simp}
 end
 
+theorem antidiagonal_eq_map_powerset [decidable_eq α] (s : multiset α) :
+   s.antidiagonal = s.powerset.map (λ t, (s - t, t)) :=
```

Comment:
```suggestion
  s.antidiagonal = s.powerset.map (λ t, (s - t, t)) :=
```
One remaining stray indent

Reply:
Damn. I hope I didn't miss another one...

Reply:
I fixed an indent in the next function at the same time. But, maybe, this one was intentional?